libvast/include/vast/arrow_table_slice_builder.hpp

```diff
@@ -3,7 +3,7 @@
 //   | |/ / __ |_\ \  / /          Across
 //   |___/_/ |_/___/ /_/       Space and Time
 //
-// SPDX-FileCopyrightText: (c) 2019 The VAST Contributors
+// SPDX-FileCopyrightText: (c) 2021 The VAST Contributors
```

Comment:
The advice from [the internet](https://matija.suklje.name/how-and-why-to-properly-write-copyright-statements-in-your-code) is to not update the date when touching a file.

> the year of the first publication – a good date would be the year in which you created the file and then [do not touch that date anymore](https://matija.suklje.name/how-and-why-to-properly-write-copyright-statements-in-your-code#why-not-bump-the-year-on-change);

Reply:
I think this was changed when I renamed it from experimental, so I probably should change it back manually.